semgrep_output_v1.atd

```diff
@@ -914,6 +920,14 @@ type dependency_parser_error = {
   ?text: string option;
 }
 
+(*****************************************************************************)
+(* Semgrep Secrets *)
+(*****************************************************************************)
+
+type historical_info = {
+  git_commit: sha1;
```

Comment:
maybe add a comment explaining what is this about; why we suddently need to commit associated with a match.

Reply:
Sure.